### ISSUE TITLE
[Library] Does the gear/settings icon do anything?

### ISSUE THREAD

**Issue by zachpoblete** [harbassan/spicetify-apps#130]
![image](https://github.com/user-attachments/assets/9f1ad820-07d2-4654-8342-e3bb84147778)

When I click it, nothing happens. Is this expected?

**Comment by harbassan** [comment]
It's not expected, but don't worry there's nothing too important there just a basic settings menu.

**Comment by harbassan** [comment]
> add snippet
> 
> #stats-app > .contentSpacing{ padding: 40px !important; } #library-app > .contentSpacing{ padding: 40px !important; }
> 
> fixes overlapping bar and settings cog

that fixes a seperate (unrelated) issue

**Comment by SamBecca16** [comment]
yeah, I doesn't do anything for me. It worked before but now it doesn't. I mean, basic settings but still I will like to make the images bigger.

Maybe if the new version? maybe a previous version would work?

I uninstalled spotify completely, but didn't fix the issue either

**Comment by harbassan** [comment]
The fix has already been made, it'll be released in the next spicetify cli version.

**Comment by ikennax** [comment]
> add snippet
> 
> #stats-app > .contentSpacing{ padding: 40px !important; } #library-app > .contentSpacing{ padding: 40px !important; }
> 
> fixes overlapping bar and settings cog

this pretty much fixed it for me
